### PR TITLE
[Feature][UI] Enable highlight and auto-complete for SQL scripts 

### LINKED ISSUE #11658: [Feature][UI] Enable highlight and auto-complete for SQL scripts
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

* Currently we have not enabled `hightlight` and `auto-complete` for SQL scripts, but it is easy to do it in the front-end.

image

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by EricGao888**
## Purpose of the pull request

* Enable highlight and auto-complete for SQL scripts.
* This PR closes: #11658 

## Brief change log

* Enable highlight and auto-complete for SQL scripts.

## Verify this pull request

* Verified by manual tests.

### CODE REVIEW

File: dolphinscheduler-ui/src/views/projects/task/components/node/fields/use-sql.ts [hunk 1]
```diff
@@ -43,6 +43,10 @@ export function useSql(model: { [field: string]: any }): IJsonItem[] {
         trigger: ['input', 'trigger'],
         required: true,
         message: t('project.node.sql_empty_tips')
+      },
+      props: {
+        language: 'sql',
+        placeholder: t('project.node.sql_statement_tips')
```
Comment (Amy0104) [inline dolphinscheduler-ui/src/views/projects/task/components/node/fields/use-sql.ts]: It seems that the monaco editor does not support placeholder yet.

Comment (EricGao888) [reply dolphinscheduler-ui/src/views/projects/task/components/node/fields/use-sql.ts]: @Amy0104 Thanks for the information. When I tested it manually, the placeholder didn't take effect. But highlight and auto-complete did work. What about removing the `placeholder` and only keeping the `language`?

Comment (Amy0104) [reply dolphinscheduler-ui/src/views/projects/task/components/node/fields/use-sql.ts]: That's better to remove the placeholder.

**Review by Amy0104 — APPROVED** [review]
LGTM

**Pull request merged**